WalletWasabi.Fluent/Validation/Validations.cs

```diff
@@ -113,15 +113,15 @@ private void UpdateAndNotify(List<ErrorDescriptor> currentErrors, List<ErrorDesc
 
 	private void OnErrorsChanged(string propertyName, List<ErrorSeverity> categoriesToNotify)
 	{
-		Func<ErrorSeverity, string> selector = x => x switch
+		static string Selector(ErrorSeverity x) => x switch
```

Comment:
@jmacato can you check if this part is correct?

Reply:
Im not sure that making it static is necessary? @yahiheb

Reply:
okay, upon more meditating on it, i think it works okay @molnard , it avoids allocs everytime OnErrorsChanged is called